### DISCUSSION TITLE
电脑网线连接到路由器lan口时，无法访问到通过交换机连接的N1设备

### DISCUSSION THREAD

**Discussion by theosoft-git** [immortalwrt/immortalwrt#1621] [Q&A]
### Describe the bug

N1刷了flippy的72+o的openwrt系统，只有一个lan口。
刷了immortalwrt的主路由的lan口连接到交换机，N1接到交换机上。
主路由关闭了dhcp服务，由N1负责局域网的DHCP。
电脑通过wifi连接到主路由或者有线连接到交换机时，一切正常。
电脑有线连接到主路由的lan口时，可以正常获取到ip地址，但是无法连接到N1（tcp任意端口/icmp都不通）。
此时ssh连接到主路由后，从主路由连接N1是正常的。其他设备通过wifi或者有线连接到交换机后，和N1的连接也是通的。
尝试把N1通交换机上拔下，接到主路由的lan口上，问题依旧。
把主路由替换成其他路由器（padavan）时没有此问题。
尝试给N1更换ip，问题依旧。
如果电脑设定的网关和dns服务器都是主路由，电脑可以正常访问因特网。内网中除N1以外的其他设备也都可以正常访问。唯独访问不到N1，但是DHCP获取ip正常。

### ImmortalWrt version

r28061-399f9a1db3

### ImmortalWrt release

23.05.4

### ImmortalWrt target/subtarget

ramips/mt7621

### Device

JDCloud RE-CP-02

### Image kind

Official downloaded image

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by ksong008** [comment]
@theosoft-git 本项目无n1支持

**Reply by theosoft-git** [reply]
我知道ImmortalWRT不支持N1，现在的问题是刷了ImmortalWRT的路由器跟N1似乎无法共存于一个网络，会导致局域网内通讯异常。总不能是ImmortalWRT特意屏蔽了与N1的通讯吧？而且是只阻隔其它设备与其通讯，路由本身却是正常的？

**Comment by theosoft-git** [comment]
这个问题仅现于官方release的版本，我拉了Immortal的源码，自己编译的版本（r28126-4180094133/23.05-SNAPSHOT）没有这个问题。
另外，发现Immortal的lan口对于百兆的网卡好像有兼容性问题，会不停的连接、断开、连接……，2～5秒就断开一次，一秒后连上。之前发现的问题可能也是这个原因（发现问题额设备是mbp和电视，都是百兆网卡）？换了个千兆的usb网卡就好了（macbook）。
